Source/Core/Core/PowerPC/GDBStub.cpp

```diff
@@ -106,7 +106,7 @@ static u8 gdb_read_byte()
   ssize_t res;
   u8 c = '+';
 
-  res = recv(sock, &c, 1, MSG_WAITALL);
+  res = recv(sock, &c, 1, GDB_MSG_WAITALL);
```

Comment:
I don't think this one is GDB specific -- it's very much a host constant.

Reply:
Right. I'll revert that.